src/main/resources/api.yaml

```diff
@@ -207,3 +312,39 @@ definitions:
     required:
       - enrollmentPeriod
       - durationPeriod
+  ProjectMemberDto:
+    type: object
+    properties:
+      userId:
+        type: integer
+        format: int64
+      userDetails:
+        $ref: '#/definitions/UserDetailsDto'
+      role:
+        $ref: '#/definitions/UserRoleDto'
+  UserRoleDto:
+    type: string
+    enum:
+      - admin
+      - member
+  UserDetailsDto:
```

Comment:
A czemu to nie ma id i nie jest po prostu `UserDto`?

Reply:
(To pytanie aktualne)

Reply:
Ponieważ założyłem, że ta struktura to nie jest pełen użytkownik tylko dane, które są wyświetlane w postaci wiersza w liście to nazwałem to `UserDetailsDto`. Mogę natomiast zmienić na `UserDto` i dodać ID.